app/assets/stylesheets/application.scss

```diff
@@ -8,3 +8,4 @@
 @import "flash_messages";
 @import "sign_in";
 @import "sign_up";
+@import "shared/wishlist";
```

Comment:
Oh, got it. It's shared between two views so I thought it should be here. If it's for global sharing only then I'll move it.